app/styles/header.less

```diff
@@ -242,12 +242,20 @@
       }
 
       .fields-list {
-        max-height: 363px;
+        //It would be better to turn this value into a dynamic value
+        //Based on the number of categories passed to the table: (n * height of each category) - search bar height
+        max-height: 488px;
 
         // shortcut not used for Safari support -_-
         overflow-x: hidden;
         overflow-y: scroll;
 
+        .cluster-name {
+          text-transform: capitalize;
+          font-weight: bold;
+          margin: @spacing-xxx-sm 0;
+        }
```

Comment:
you could use some classes on the html side : `class="text-style-capitalize text-style-bold margin-top-xxx-sm margin-bottom-xxx-sm"`